astroplan/core.py

```diff
@@ -158,6 +157,52 @@ def __init__(self, location=None, timezone='UTC', name=None, latitude=None,
             raise TypeError('timezone keyword should be a string, or an '
                             'instance of datetime.tzinfo')
 
+    def astropy_to_local_time(self, astropy_time):
+        '''
+        Convert the `~astropy.time.Time` object ``astropy_time`` to a
+        localized `~datetime.datetime` object.
+
+        Parameters
+        ----------
+        time : `~astropy.time.Time`
```

Comment:
Would you like that? I was using `time` only once because it's already a long name, but I can stick it in.